src/include/daos_srv/bio.h

```diff
@@ -598,6 +611,18 @@ bio_yield(void)
 int bio_get_dev_state(struct nvme_health_stats *dev_state,
 		      struct bio_xs_context *xs);
 
+/*
+ * Helper function to get the internal blobstore state for a given xstream.
+ * Used for daos_test validation in the daos_mgmt_get_bs_state() C API.
+ *
+ * \param dev_state	[OUT]	BIO blobstore state
+ * \param xs		[IN]	xstream context
+ *
+ * \return			Zero on success, negative value on error
```

Comment:
Does no return anything so this line is not needed?

Reply:
good catch